src/RoboFile/RoboFileInterface.php

```diff
@@ -43,4 +43,26 @@ public function runTestSuites(
 	 * @since   3.7.0
 	 */
 	public function prepareTestingPackage($params = array('dev' => false));
+	/**
+	 * Function for actual execution of a individual test of this extension
+	 *
+	 * @param   array  $opts  Array of configuration options:
+	 *                        - 'env': set a specific environment to get configuration from
+	 *                        - 'debug': executes codeception tasks with extended debug
+	 * 						  - 'suite':
+	 * 						  - 'test': example: 'install/InstallWeblinksCest.php/installWeblinks'
+	 *
+	 * @return void
+	 *
+	 * @since   x.x.x
+	 */
+	public function runTestSingle(
```

Comment:
The actual reason I did this, is because it has a different purpose that runTestSuites. 
https://github.com/joomla-projects/gsoc17_parallel_testing/blob/container-test/tests/RoboFile.php#L81-L105
And for me it makes sense to have a command that runs all the tests (`runTestSuites`), and one that runs a single test. But I can make the check (https://github.com/isacandrei/joomla-testing-robo/blob/6d648ae043192b4e26974f5353be8a9d308d01c7/src/RoboFile/ClientContainer.php#L49-L56) inside `runTestSuites`, in order to not change the interface.